src/main/java/org/reactome/qa/doisuggester/DOISuggesterMain.java

```diff
@@ -0,0 +1,172 @@
+package org.reactome.qa.doisuggester;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.QuoteMode;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.reactome.qa.doisuggester.Suggester.Suggestion;
+import org.reactome.util.general.MandatoryProperties;
+import org.reactome.util.general.MandatoryProperties.PropertyHasNoValueException;
+import org.reactome.util.general.MandatoryProperties.PropertyNotPresentException;
+
+public class DOISuggesterMain
+{
+
+	public static void main(String[] args) throws IOException
+	{
+		// TODO: Use log4j instead of printing to the console. Will come in a future PR.
+		System.out.println("Checking for RLEs that might need a DOI...");
+		String pathToConfig = Paths.get("src", "main", "resources", "config.properties").toString();
+		String username;
+		String password;
+		String database;
+		String databasePrev;
+		String host;
+		int port;
```

Comment:
I think all these can be declared within the try block when they are assigned from the props object?